app/src/main/java/swati4star/createpdf/util/InvertPdf.java

```diff
@@ -0,0 +1,118 @@
+package swati4star.createpdf.util;
+
+import android.graphics.Bitmap;
+import android.os.AsyncTask;
+import android.os.ParcelFileDescriptor;
+import com.itextpdf.text.Rectangle;
+import com.itextpdf.text.pdf.GrayColor;
+import com.itextpdf.text.pdf.PdfContentByte;
+import com.itextpdf.text.pdf.PdfGState;
+import com.itextpdf.text.pdf.PdfReader;
+import com.itextpdf.text.pdf.PdfStamper;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+
+import swati4star.createpdf.interfaces.OnPDFCreatedInterface;
+
+import static android.os.ParcelFileDescriptor.MODE_READ_ONLY;
+
+public class InvertPdf extends AsyncTask<Void, Void, Void> {
+    private String mPath;
+    private OnPDFCreatedInterface mOnPDFCreatedInterface;
+    private ArrayList<Bitmap> mBitmaps;
+    private StringBuilder mSequence;
+    private Boolean mIsNewPDFCreated;
+
+    public InvertPdf(String mPath, OnPDFCreatedInterface onPDFCreatedInterface) {
+        this.mPath = mPath;
+        mSequence = new StringBuilder();
+        mBitmaps = new ArrayList<>();
+        this.mOnPDFCreatedInterface = onPDFCreatedInterface;
+    }
+
+    @Override
+    protected void onPreExecute() {
+        super.onPreExecute();
+        mOnPDFCreatedInterface.onPDFCreationStarted();
+        mIsNewPDFCreated = false;
+    }
+
+
+    @Override
+    protected Void doInBackground(Void... voids) {
+        ParcelFileDescriptor fileDescriptor = null;
+        try {
+            if (mPath != null)
+                // resolve pdf file path based on relative path
+                fileDescriptor = ParcelFileDescriptor.open(new File(mPath), MODE_READ_ONLY);
+
+
+            if (fileDescriptor != null) {
+                String outputPath = mPath.replace(".pdf", "_inverted" + ".pdf");
```

Comment:
use pdfExtension variable instead of ".pdf"
https://github.com/Swati4star/Images-to-PDF/blob/a1e06344e81bd49a39cf5bd2ad0a8c826e505b61/app/src/main/java/swati4star/createpdf/util/Constants.java#L47